package.json

```diff
@@ -24,16 +24,21 @@
     "discord.js": "^12.5.1",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "ffmpeg-static": "^4.3.0",
     "moment": "^2.29.1",
     "mongodb": "^3.5.9",
+    "node-opus": "^0.3.3",
     "typescript": "^4.0.5",
     "winston": "^3.3.3",
-    "winston-papertrail": "^1.0.5"
+    "winston-papertrail": "^1.0.5",
+    "yt-search": "^2.8.0",
+    "ytdl-core": "^4.5.0"
```

Comment:
is this stuff against yt terms of service?

Reply:
didnt think about that, after reading about it, it seems very questionable Sadge
never thought about that